pkg/tfvars/azure/azure.go

```diff
@@ -27,27 +28,48 @@ type config struct {
 	ImageURL                    string            `json:"azure_image_url,omitempty"`
 	Region                      string            `json:"azure_region,omitempty"`
 	BaseDomainResourceGroupName string            `json:"azure_base_domain_resource_group_name,omitempty"`
+	NetworkResourceGroupName    string            `json:"azure_network_resource_group_name,omitempty"`
+	VirtualNetwork              string            `json:"azure_virtual_network,omitempty"`
+	ControlPlaneSubnet          string            `json:"azure_control_plane_subnet,omitempty"`
+	ComputeSubnet               string            `json:"azure_compute_subnet,omitempty"`
+	PreexistingNetwork          bool              `json:"azure_preexisting_network,omitempty"`
+}
+
+// TFVarsSources contains the parameters to be converted into Terraform variables
+type TFVarsSources struct {
+	Auth               Auth
+	Platform           *azuretypes.Platform
```

Comment:
this field should not be added, as it didn't exist before in the TFVars variable list.